spec/models/user_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+describe User do
+
+    describe "needs_shave_or_trimming" do
```

Comment:
Will fix the spacing in code indentation where ever required to make use of two spaces for indentation instead of 4.